src/App.css

```diff
@@ -8,58 +27,272 @@
 
 .App nav {
   display: flex;
-  justify-content: space-around;
-
-  /* background: linear-gradient(45deg,#ff0084, #33001b); */
+  justify-content: flex-start;
+  background-color: white;
+  padding: 1rem;
+  border-bottom: 1px solid #ccc;
+  gap: 2rem;
+  padding-left: 4rem;
+  align-items: center;
+}
 
-  /* color: white; */
+nav h1 {
+  color: #0290ff;
 }
 
 .App nav ul {
   display: flex;
   justify-content: flex-start;
   align-items: center;
-  gap: 1rem;
+  gap: 2rem;
 }
 
 .App nav ul li a {
   text-decoration: none;
+  color: #ccc;
+  font-weight: 400;
+  font-size: 1.2rem;
+}
+
+.App nav ul li a:focus {
+  color: #7f7f7f;
+}
+
+nav div.user-icon {
+  color: #0290ff;
+  display: inline-flex;
+  align-items: center;
+  margin-left: auto;
+  margin-right: 4rem;
+}
+
+nav div.user-icon .fa-icon {
+  width: 2rem;
+  height: 2rem;
+}
+
+.App > ul.book-list {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin: 0 4rem;
+}
+
+li.book {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  gap: 1rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border: 1px solid #ccc;
+  border-radius: 3px;
+  background-color: white;
+}
+
+li.book .book-detail {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 30rem;
+}
 
-  /* color: white; */
+li.book .book-detail .book-title {
+  align-self: flex-start;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+li.book .line {
+  width: 0.055rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
+  border: solid 1px #e8e8e8;
+  background-color: #e8e8e8;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+li.book .book-progress {
+  display: flex;
+  gap: 1rem;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.book-progress .text-peogress {
   display: flex;
   flex-direction: column;
+  align-items: flex-start;
+}
+
+.book-chapter {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 1rem;
+}
+
+li.book .book-chapter .update-progress {
+  width: 11.5rem;
+  height: 2.063rem;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  outline: none;
+  border: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  color: #fff;
+  font-family: 'RobotoSlab', sans-serif;
+}
+
+.book-chapter .current-chapter {
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```

Comment:
- [ ] Please kindly look through the lines of your code carefully, I noticed you're repeating this lines of code a lot. Can we create a `class` for it and apply it to the appropriate elements? This will ensure will follow CSS DRY principle.

Reply:
[Fixex here](https://github.com/ZewdieMc/book-store/pull/5/commits/6073db325beef4510e5c05d0dc4027d3d9aed9a2)